baybe/kernels/__init__.py

```diff
@@ -1,11 +1,29 @@
 """Kernels for Gaussian process surrogate models."""
```

Comment:
We should put a link to the gpytorch documentation here for more details on the kernels: Currently, if you do not already know the kernels, you have no way of understanding what they actually do. ALternatively, we might want to adjust the docstrings for all of the kernels to have our documentation more self-contained, don't know what is the better option.

Reply:
added link in kernels and prior top level page